easybuild/easyconfigs/t/TINKER/TINKER-8.7.2-foss-2019b.eb

```diff
@@ -0,0 +1,21 @@
+name = 'TINKER'
+version = '8.7.2'
+
+homepage = 'https://dasher.wustl.edu/tinker'
+description = """The TINKER molecular modeling software is a complete and general package for molecular mechanics
+ and dynamics, with some special features for biopolymers."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://dasher.wustl.edu/tinker/downloads/']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['%(name)s-%(version)s_fix_BAR_mode_1.patch']
+checksums = [
+    'f9e94ae0684d527cd2772a4a7a05c41864ce6246f1194f6c1c402a94598151c2',  # tinker-8.7.2.tar.gz
+    'c5122fe5ed778ceacf9bad144c5eb381506cab407a7733a6de05b4252bd1e9cf',  # TINKER-8.7.2_fix_BAR_mode_1.patch
+]
+
+runtest = False
```

Comment:
why the change to skipping tests?

Reply:
As far as I remember one of the tests simply never managed to finish. It just ran for hours and hours and ...

Reply:
can you add a comment about that?